Yosemite/YosemiteTests/Stores/PaymentGatewayStoreTests.swift

```diff
@@ -0,0 +1,86 @@
+import XCTest
+import TestKit
+
+@testable import Yosemite
+@testable import Networking
+@testable import Storage
+
+/// PaymentGatewayStore Unit Tests
+///
+final class PaymentGatewayStoreTest: XCTestCase {
```

Comment:
super nit: to be consistent with the file name is `PaymentGatewayStoreTests` and other tests 😄 
```suggestion
final class PaymentGatewayStoreTests: XCTestCase {
```